### DISCUSSION TITLE
Can the UniFi Cloud API Authorize Guest WiFi Users?

### DISCUSSION THREAD

**Discussion by gstratford** [Art-of-WiFi/UniFi-Cloud-API-client#2] [General]
Hi everyone,

I’ve been using the UniFi-API-client to manage guest WiFi users on our self-hosted UniFi Controller, and it’s been working great. Now, we have some devices on the UniFi Cloud platform, and I’m wondering if it’s possible to authorize guest users through the unifi-cloud-api-client like I can with the self-hosted version.

Has anyone successfully done this or knows if the cloud API supports guest user authorization? I didn’t see anything definitive in the docs, so I’m hoping to confirm from the community.

Thanks in advance for any insights!

Best, Greg

**Comment by malle-pietje** [comment]
@gstratford the new UniFi “Cloud” API is pretty limited at the moment (it is at version 0.1) and for now only supports listing of sites, hosts, and devices. For some large environments this already can be very useful.

It would be ideal if this API can also be used for other things such as guest device authorizations etc. but we’ll have to which direction Ubiquiti is heading.

If anyone from the Ubiquiti team is reading this and is willing and able to respond here that would be great🤞👍

**Reply by gstratford** [reply]
@malle-pietje - Thanks for the info/perspective. Yes, it would be great if the Cloud API headed towards the full featured functionality of the controller api. It would certainly expand the number of cloud devices our organization would deploy :)

**Reply by malle-pietje** [reply]
Agreed👍
